api/examples/python/README.md

```diff
@@ -8,16 +8,13 @@ notifications.
 First, install the dependencies.
 
 ```
-sudo /usr/bin/python -m pip install websocket-client
-sudo /usr/bin/python -m pip install protobuf
-
+mkvirtualenv iterm2api
```

Comment:
mkvirtualenv is a fairly common wrapper around python's virtualenv tools: 
https://virtualenvwrapper.readthedocs.io/en/latest/command_ref.html

You can get the same effect with
```
mkdir -p ~/.virtualenv
cd ~/.virtualenv
virtualenv xxx
source ./xxx/bin/activate
```